schema/postgres/cadence/versioned/v0.6/extend_workflow_id_length.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE executions ALTER workflow_id TYPE text;
```

Comment:
Can you also update cadence/schema.sql and visibility/schema.sql to reflect these changes? During first time setup we run only the schema files, the versions are used only when an existing cluster is upgraded. The schema files should match the outcome of running all the version upgrades.